cypress/e2e/pages/index.spec.ts

```diff
@@ -215,30 +225,33 @@ context("/ on iphone-x", () => {
 
   it("should have LiquidityPools", () => {
     cy.findByTestId("LiquidityPoolList").should("be.visible");
+    cy.findAllByTestId("LiquidityPoolCard").each(($el) => {
+      cy.wrap($el).within(() => {
+        cy.findByTestId("LiquidityPoolCard.PoolPairSymbol").should(
+          "be.visible"
```

Comment:
Can we make this into a single line like this `.should("be.visible");` to maintain its readability